vortexasdk/api/vessel.py

```diff
@@ -76,7 +95,7 @@ class VesselEntity(IDName):
     end_timestamp: Optional[ISODate] = None
     fixture_id: Optional[str] = None
 
-    scrubber: Optional[Scrubber] = None
-    flag: Optional[Flag] = None
+    scrubber: Optional[List[Scrubber]] = Field(default_factory=list)
```

Comment:
I know you are treating this as interchangeable with `None` but I would prefer to be consistent and use the same approach.

Reply:
Fixed.